src/components/LetterLoad.vue

```diff
@@ -83,6 +92,15 @@ export default {
 
       return ''
     }
+  },
+  watch: {
+    customMessage(newMessage) {
+      console.log(newMessage)
+      this.$store.commit('setGenericValue', { key: 'userCustomMessage', value: newMessage })
+    }
+  },
+  persistCustomization(){
```

Comment:
Unless I am missing it somewhere, this is not used anywhere now and we can delete it.

Reply:
Thanks for checking this Glenn. I have deleted the persistCustomization() function.